lib/open_food_network/sales_tax_report.rb

```diff
@@ -9,9 +9,27 @@ def initialize(user, params)
     end
 
     def header
-      ["Order number", "Date", "Items", "Items total (#{currency_symbol})", "Taxable Items Total (#{currency_symbol})",
-        "Sales Tax (#{currency_symbol})", "Delivery Charge (#{currency_symbol})", "Tax on Delivery (#{currency_symbol})", "Tax on Fees (#{currency_symbol})",
-        "Total Tax (#{currency_symbol})", "Customer", "Distributor"]
+      case params[:report_type]
+      when "tax_types"
+        [I18n.t(:report_header_order_number),
+         I18n.t(:report_header_date),
+         I18n.t(:report_header_items),
+         I18n.t(:report_header_items_total, currency_symbol: currency_symbol),
+         I18n.t(:report_header_taxable_items_total, currency_symbol: currency_symbol),
+         I18n.t(:report_header_sales_tax, currency_symbol: currency_symbol),
+         I18n.t(:report_header_delivery_charge, currency_symbol: currency_symbol),
+         I18n.t(:report_header_tax_on_delivery, currency_symbol: currency_symbol),
+         I18n.t(:report_header_tax_on_fees, currency_symbol: currency_symbol),
+         I18n.t(:report_header_total_tax, currency_symbol: currency_symbol),
+         I18n.t(:report_header_customer),
+         I18n.t(:report_header_distributor)]
+      else
+        [I18n.t(:report_header_order_number),
+         I18n.t(:report_header_total_excl_vat, currency_symbol: currency_symbol)] +
+        relevant_rates.map { |rate| "%s (%.1f%%) (%s)" % [rate.name, rate.amount.to_f * 100, currency_symbol] } +
```

Comment:
@oeoeaio sorry I had not seen you pinged me, @ltrls just pointed it to me. Actually as you say for the moment it is super unlikely that an instance use both as the super admin has to choose if "taxes are included in price or not" for the whole instance. But we are anticipating for the tax overhaul :-) So it might be that the master price includes VAT for some products and that it does not for others. That doesn't matter for the tax display, we want the whole accumulated tax in the same column, we want for the order basically to know what is the total amount of VAT collected. Hope it's clear :) @ltrls let me know if you don't understand also :-)